clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -3523,6 +3525,23 @@ static bool InvalidWorkGroupSizeAttrs(const Expr *MGValue, const Expr *XDim,
            ZDimExpr->getResultAsAPSInt() != 1));
 }
 
+// If the [[intel::max_work_group_size(X, Y, Z)]] attribute is specified on
+// a declaration along with [[sycl::reqd_work_group_size(X1, Y1, Z1)]]
+// attribute, check to see if values of reqd_work_group_size arguments are
+// equal or less than values of max_work_group_size attribute arguments.
+static bool checkWorkGroupSizeAttrValues(const Expr *LHS, const Expr *RHS) {
```

Comment:
Given that this routine is specific to checking reqd_work_group_size and max_work_group_size arguments, I would suggest a little easier name for the parameters like:
```suggestion
static bool checkWorkGroupSizeAttrValues(const Expr *RWGS, const Expr *MWGS) {
```